mscore/scoreview.cpp

```diff
@@ -4273,6 +4276,15 @@ void ScoreView::cmdAddText(Tid tid, Tid customTid)
                   if (!cr)
                         break;
                   s = new StaffText(_score, Tid::STAFF);
+                  // if the previous text is UNSTYLED or NOSTYLE
+                  // and the next text is created using text tab command
+                  // (only those created in this way can make pf not STYLED, see definition of ScoreView::textTab())
+                  // it inherits the same PropertyFlags and Placement
+                  // making it much easier to put all texts above/below staves
+                  if (pf != PropertyFlags::STYLED) {
```

Comment:
Can the `ScoreView::cmdAddText` method be used for other cases now or in the future?  
My concern is that the correction and commentary is for one use case, but the “know nothing” method of how it is used. And in other use cases, this behavior will not be expected.

Reply:
Don't get you. The line to which you attached this comment is only appliable to `case Tid::STAFF`, isn't it?

Reply:
I investigated for myself in more detail, I remove my question :)